ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,9 @@
+<!--
+Please consider including the answers below when relevant:
+
+How do you use Sentry:
+Sentry SaaS (sentry.io) or Self hosted (open-source, which version?)
+
+What SDK and version?
+e.g: JavaScript 5.11.1, .NET 1.2.0
```

Comment:
This is good to have, though if an issue is opened in a `sentry-foo` repo, I'd assume the SDK to be "foo".

Reply:
Right. I tried to use two lines and little text but this means. SDK name **and version**. But sure it's where applicable. As @mattrobenolt said, on our internal repos and other non-SDKs that'll be useless.